src/main/scala/com/microsoft/partnercatalyst/fortis/spark/dto/FortisItem.scala

```diff
@@ -0,0 +1,7 @@
+package com.microsoft.partnercatalyst.fortis.spark.dto
+
+trait FortisItem {
```

Comment:
Why do we need interfaces for which there will only ever be one implementation? We can always introduce the interface at the point at which we'll need it. Less code is always better as it's less overhead to understand and maintain; [yagni](https://martinfowler.com/bliki/Yagni.html). Same applies to `TransformContext` below.